src/applications/facility-locator/components/VABenefitsCall.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { parsePhoneNumber } from '../utils/phoneNumbers';
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
+
+const { formattedPhoneNumber, extension, contact } = parsePhoneNumber(
+  '800-827-1000',
+);
+
+export default function VABenefitsCall() {
+  return (
+    <div>
+      <p className="p1">
+        <strong>To check on the status of your claim or appeal</strong>, use our{' '}
+        <a href="/claim-or-appeal-status">online claim status tool</a>.
+      </p>
+      <p className="p1">
+        <strong>To get help with benefits</strong>, call{' '}
+        <Telephone contact={contact} extension={extension}>
+          {' '}
+          {formattedPhoneNumber}
+        </Telephone>{' '}
```

Comment:
Please note that the Telephone component will throw an Error if the contact is < 10 characters long.

I have filed an [issue](contact) to have that removed, but in the meantime, you may want to do some error checking, or add an [ErrorBoundary](https://reactjs.org/docs/error-boundaries.html) component.